test/remote/gateways/remote_authorize_net_cim_test.rb

```diff
@@ -263,14 +282,14 @@ def test_successful_get_customer_profile_with_multiple_payment_profiles
 
     assert response.test?
     assert_success response
-    assert_nil response.authorization
+    assert_equal @customer_profile_id, response.authorization
     assert customer_payment_profile_id = response.params['customer_payment_profile_id']
     assert customer_payment_profile_id =~ /\d+/, "The customerPaymentProfileId should be numeric. It was #{customer_payment_profile_id}"
 
     assert response = @gateway.get_customer_profile(customer_profile_id: @customer_profile_id)
     assert_equal 2, response.params['profile']['payment_profiles'].size
-    assert_equal 'XXXX4242', response.params['profile']['payment_profiles'][0]['payment']['credit_card']['card_number']
-    assert_equal 'XXXX1234', response.params['profile']['payment_profiles'][1]['payment']['credit_card']['card_number']
+    assert_equal 'XXXX1234', response.params['profile']['payment_profiles'][0]['payment']['credit_card']['card_number']
```

Comment:
I am not sure if this order is deterministic or not, but every time we ran this, they were reversed.

Reply:
Seems brittle to depend on that though, any way we can easily identify and find the profile? Seeing as these remote tests are run locally only, it's not great if it ends up breaking in the future, it won't be fun to debug.